### PR TITLE
Improve LaTeX section

### PR DESCRIPTION
**Pull request by Kyslik**
- updated MacTex and BasicTex sections
- added note about tlmgr
- added VS Code as editor with a link for extension
- added note about PDF readers

### CODE REVIEW

File: LaTeX/README.md [hunk 1]
```diff
@@ -1,25 +1,32 @@
 # LaTeX
```
Comment (simeg) [inline LaTeX/README.md]: Didn't know about MD022, but I've been thinking about styling/linting for this project. Maybe we should look into setting it up as part of the CI process?

File: LaTeX/README.md [hunk 2]
```diff
@@ -34,3 +37,6 @@ Now you should pick a LaTeX editor, here are some to choose from:
   5. [LyX](http://www.lyx.org/)
   6. [TeXlipse](http://texlipse.sourceforge.net/)
   7. [Sublime Text 3 - LaTeXTools](https://github.com/SublimeText/LaTeXTools)
+  8. [Visual Studio Code - LaTeX-Workshop](https://github.com/James-Yu/LaTeX-Workshop)
+
+> **Note**: You may find useful a PDF reader that watches your PDF files, like Skim `brew cask install skim` (Preferences -> Sync).
```
Comment (simeg) [inline LaTeX/README.md]: This sentence may a small edit, I think a word is missing.

File: LaTeX/README.md (hunk 1 above)
Comment (simeg) [reply LaTeX/README.md]: I created #239 for this.

Comment (Kyslik) [reply LaTeX/README.md]: Yea, I use VSC now, and have https://github.com/DavidAnson/vscode-markdownlint installed.

File: LaTeX/README.md (hunk 2 above)
Comment (Kyslik) [reply LaTeX/README.md]: @simeg Not sure how to word it better in order to make the intent clear.

> You may find useful a PDF reader that watches your PDF files, like Skim (can be installed by `brew cask install skim` and Sync feature can be enabled by (Preferences -> Sync)).

Comment (simeg) [reply LaTeX/README.md]: Ah, I see now why I thought it was missing a word.

I find this sentence hard to read:
>You may find useful a PDF reader [..]

Could we re-word it to something like this?
>You may want to use a PDF reader [..]

Sorry for nitpicking.

Comment (Kyslik) [reply LaTeX/README.md]: No big deal :), thank you.

**Pull request merged**